pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "scikit_build_core.build"
 
 [project]
 name = "MCEq"
-version = "1.3.0"
+version = "1.3.5"
```

Comment:
whats our policy here regarding versioning? how many numbers to bump in general and why 5 here? 😊